controllers/stats.js

```diff
@@ -314,6 +350,10 @@ const get_plays_by_account_daily = asyncHandler(async (req, res, next) => {
     .andWhere("play.created_at", ">", d30)
     .groupBy([groupBy, "artist.user_id"])
     .then((data) => {
+      if (data.length === 0) {
+        res.send({ success: true, data: data });
```

Comment:
should this one also have a zero `playTotal` value?
```suggestion
        res.send({ success: true, data: { playTotal: 0 } });
```

Reply:
nope, nevermind, i'm looking at the wrong one i think

Reply:
No this one is supposed to return an array of play totals per day